### PR TITLE
Add config option templates.index, set to current default path, & use…

### PR DESCRIPTION
**Pull request by aljones15**
Features:

1. Add a new config option `templates.index`
2. Set `templates.index` to the current template
3. Add minor changelog entry

Tests:
1. continues to work even if a custom template is not set
2. works with a custom template
3. throws if the custom template path does not exist

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # bedrock-webpack ChangeLog
 
+## 9.1.3
```
Comment (davidlehn) [inline CHANGELOG.md:3]:
```suggestion
## 9.1.0 - 2023-xx-xx
```

**Review by davidlehn — COMMENTED** [review]
- Looks like this would work as a hammer to solve full replacement of the index HTML.  The problem with that is keeping updates in sync (which has been a rare thing).
- Maybe need to list out the goals?  There might be a better solution.
- I'm forgetting if this is processed as a template now and how.  We used to do much more with template processing.  But now it's all client side or HTML Webpack Plugin defaults.
- I always assumed we'd jump more into HTML Webpack Plugin use.  There are *lots* of plugins available.  Other projects use (or did in the past) that project and plugins for far more than we are using it for at the moment.  There might be a plugin that does the modifications you were going to do.

**Comment by aljones15** [comment]
@davidlehn ok so it looks like we could use a `html-webpack-plugin` option: https://github.com/jantimon/html-webpack-plugin#options

Mostly we could make `meta` configurable.

this would require adding a new config option which would be something like `config['bedrock-webpack'].htmlWebPackPluginConfig = {meta: 'viewport-fit': 'cover'}` or something like that.

Code to be modified is the same as in this PR:

https://github.com/digitalbazaar/bedrock-webpack/blob/5d4834f620392f7bcb6c27710b2b38d79c7e1a30/lib/index.js#L244-L246

This might be better, but it also might be accomplishable using the existing `config['bedrock-webpack'].configs.push({...})`

How do you want to handle this?

**Comment by aljones15** [comment]
related: https://github.com/digitalbazaar/bedrock-webpack/pull/52

**Comment by aljones15** [comment]
closing as we're using a javascript only solution.

**Pull request closed without merging**